packages/models-library/src/models_library/api_schemas_rpc_async_jobs/async_jobs.py

```diff
@@ -1,14 +1,13 @@
 from datetime import datetime
 from typing import Any, TypeAlias
-from uuid import UUID
 
 from models_library.users import UserID
 from pydantic import BaseModel, model_validator
 from typing_extensions import Self
 
 from ..progress_bar import ProgressReport
 
-AsyncJobId: TypeAlias = UUID
+AsyncJobId: TypeAlias = str
```

Comment:
TIP: I recommend you using `IDStr`. It is a constraint string for identifiers that are strings.

Reply:
It's not gonna work. I will unify the identifier with a more properly typed when I do the final integration